fdbserver/workloads/MetaclusterManagementWorkload.actor.cpp

```diff
@@ -173,13 +173,27 @@ struct MetaclusterManagementWorkload : TestWorkload {
 						retried = true;
 					}
 				} catch (Error& e) {
-					if (e.code() == error_code_cluster_already_exists && retried && !dataDb->registered) {
+					state Error registerError = e;
+					if (registerError.code() == error_code_cluster_removed ||
+					    registerError.code() == error_code_cluster_not_empty) {
+						if (e.code() == error_code_cluster_removed) {
```

Comment:
This wasn't really intentional, but it doesn't make much difference here because they are both the same. I can change this to use `registerError` throughout.